lib/services/orchestrator.js

```diff
@@ -198,7 +196,7 @@ Orchestrator.prototype.stabilised = Promise.promisify(function (callback) {
       error.name = 'StabiliseTimeout';
 
       clearInterval(_this.stableReportInterval);
-      while (callback = _this.stabiliseWaiting.shift()) callback(error);
+      while (callback = _this.stabiliseWaiting.shift()) callback(error); // jshint ignore:line
```

Comment:
rather add rules to .jshintrc

Reply:
Adjusted the offending code instead. On master. Had already merged.